File: app/src/main/java/com/imangazalievm/bubbble/presentation/mvp/global/permissions/PermissionsManager.java
package com.imangazalievm.bubbble.presentation.mvp.global.permissions;

public interface PermissionsManager {

    void requestPermission(Permission permission, PermissionRequestListener listener);

    void requestPermissions(Permission[] permissions, PermissionRequestListener listener);

    boolean checkPermissionGranted(Permission permission);

}

File: app/src/main/java/com/imangazalievm/bubbble/presentation/global/permissions/PermissionsManager.kt
package com.imangazalievm.bubbble.presentation.global.permissions;

public interface PermissionsManager {

    void requestPermission(Permission permission, PermissionRequestListener listener);

    void requestPermissions(Permission[] permissions, PermissionRequestListener listener);

    boolean checkPermissionGranted(Permission permission);

}
